detection-rules/attachment_docusign_image_suspicious_links.yml

```diff
@@ -36,6 +36,10 @@ source: |
                                     "(wichtig|dringend|sofort)"
                  )
                )
+               and not (
+                   strings.ilike(.scan.ocr.raw, "*DocuSigned By*") and 
+                   strings.ilike(.scan.ocr.raw, "*DocuSign Envelope ID*")
+              )
```

Comment:
is there anything we could use that might be harder to bypass, like docusign in the headers or exiftools? if not, i think we could roll with this

Reply:
I wish... I looked at the sample and didn't find anything. I think we'll cross that bridge if we get there.